Update scaffolding etc.

chrzyki opened the pull request:
```
============================= test session starts ==============================
platform darwin -- Python 3.6.3, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
rootdir: wold2, inifile: setup.cfg
plugins: mock-1.7.1, cov-2.5.1, clld-0.4.0
collected 28 items
tests/test_functional.py                      [ 96%]
tests/test_selenium.py ...........................                                                 [100%]
---------- coverage: platform darwin, python 3.6.3-final-0 -----------
Name                            Stmts   Miss  Cover   Missing
-------------------------------------------------------------
wold2/__init__.py                  34      0   100%
wold2/adapters.py                  42     10    76%   68-73, 76, 110-112
wold2/assets.py                     5      0   100%
wold2/datatables.py               173      1    99%   62
wold2/interfaces.py                 2      0   100%
wold2/maps.py                      48      0   100%
wold2/models.py                    86      0   100%
wold2/scripts/__init__.py           0      0   100%
wold2/scripts/initializedb.py     154    154     0%   1-376
wold2/util.py                      71      3    96%   18-19, 151
wold2/views.py                      0      0   100%
-------------------------------------------------------------
TOTAL                             615    168    73%


========================== 28 passed in 29.70 seconds ==========================.
```

```
  py27: commands succeeded
  py34: commands succeeded
  py35: commands succeeded
  py36: commands succeeded
  congratulations :)
```

xrotwang commented on setup.cfg:
```diff
@@ -6,23 +6,29 @@ addopts =
 [extract_messages]
 add_comments = TRANSLATORS:
-output_file = wold2/locale/wold.pot
+output_file = wold2/locale/wold2.pot
```
It may have been unfortunate to call the file `wold.pot` to begin with, but renaming it lateron is typically not a good idea, because the old one does exist after all.

chrzyki replied:
Alright, was probably a little bit overeager to put my newfound knowledge about re-creating the translation files into practice ... will change!

xrotwang replied:
It seems you didn't run `compile_catalog`, though, because `clld.mo` isn't changed. So all of this would not have had an effect. Maybe the lesson is that apps with translations should have at least one functional test making sure translations work.

xrotwang commented on wold2/locale/en/LC_MESSAGES/clld.po:
```diff
@@ -1,46 +1,48 @@
+#: wold2/__init__.py:15
 msgid "Contribution"
-msgstr "Vocabulary"
+msgstr ""
```
So it seems as if this file has been recreated, thereby losing the existing translations - possibly due to renaming the catalog file?

chrzyki commented:
Reverted the changes. As far as I can tell GitHub is probably going to track these files as changed, even though I reverted the changes (and I don't know how to revert back in this PR without marking the files as having been changed). If this is bothersome, I could just do a fresh PR.
